api/app/src/external/commonwell/document.ts

```diff
@@ -22,22 +29,73 @@ export type DocumentWithLocation = Required<Pick<Document, "id">> &
   };
 
 export async function getDocuments({
-  cxId,
-  patientId,
+  patient,
   facilityId,
 }: {
-  cxId: string;
-  patientId: string;
+  patient: Patient;
+  facilityId: string;
+}): Promise<DocumentReference[]> {
+  try {
+    const cwDocuments = await internalGetDocuments({ patient, facilityId });
+
+    const documents = cwDocuments.map(toDomain(patient));
+
+    return await createOrUpdate(patient, documents);
+  } catch (err) {
+    console.log(`Error: `, err);
+    capture.error(err, {
+      extra: {
+        context: `cw.queryDocuments`,
```

Comment:
nit: any reason we're using template strings in all of these places?

Reply:
not a template, just a way I found useful to indicate the origin of the error using a sort of namespace (from higher level/noun to lower level/verb)